examples/native/ios/Podfile.lock

```diff
@@ -415,6 +418,8 @@ EXTERNAL SOURCES:
     :path: "../node_modules/react-native/Libraries/Vibration"
   ReactCommon:
     :path: "../node_modules/react-native/ReactCommon"
+  RNDateTimePicker:
```

Comment:
is this the breaking change?

Reply:
It will be yes because when using on-device knobs you would need to manually install the date time library via npm and then run pod install (iOS) so that the native library is available. If you were to upgrade without following these steps it would give you errors. I want to see if there is a way I can avoid this but not sure it's possible. Maybe something similar to the asyncStorage solution is possible, where you can disable it if you don't use it.